src/solver/type_getters.jl

```diff
@@ -586,6 +586,11 @@ function get_simulation(config::AtmosConfig, comms_ctx)
     output_dir = isnothing(out_dir) ? default_output : out_dir
     mkpath(output_dir)
 
+    if parsed_args["log_params"]
+        logfilepath = joinpath(output_dir, "$(job_id)_$FT.toml")
+        CP.log_parameter_information(toml_dict, logfilepath)
+    end
+
```

Comment:
I would have put this directly in the `get_integrator`. I am not sure if logging the parameters logically belongs to to the `get_simulation` function. 

Also, maybe we can have the word `param`(eters) in the name of the output file